Sources/Widgets/Widgets3D/LineWidget/index.js

```diff
@@ -185,6 +186,7 @@ const DEFAULT_VALUES = {
   handle1Shape: HandleRepresentationType.ARROWHEAD6,
   handle2Shape: HandleRepresentationType.ARROWHEAD6,
   positionOnLine: 0.5, // Position of the text on the line where 0 is handle1 and 1 is handle2
+  text: 'Text orginal',
```

Comment:
by default it should either empty. You must add a comment to give a clue on how to make it display the actual distance between handle1 and handle2

Reply:
I'm actually confused on what the difference is between this text and the one in the text handle. Please comment